bitmap-annashivwill/test/additional-tests.js

```diff
@@ -0,0 +1,144 @@
+        done();
+      });
+    });
+  });
+});
+
+
+///// GREYSCALE TRANSFORMATION TESTS /////
+describe('GREYSCALE TRANSFORMATION MODULE --', function() {
+  describe('Should return a failed test:', function() {
+    it('for incorrect filepath of GREYSCALE.', function(done) {
+      fileReader(`${__dirname}/../img/notgrey.bmp`, function(err) {
+        expect(err).to.be.an('error');
+        done();
+      });
+    });
+  });
+  describe('Should return a passing test:', function() {
+    it('for the correct transformation for GREYSCALE.', function(done) {
+      fileReader(`${__dirname}/../img/grey.bmp`, function(err, data) {
+        expect(err).to.equal(null);
+        // expect(data.toString('hex')).to.be.a('string');
+        expect(data.toString('hex').slice(54, 1078)).to.equal('0008000000000010270000120b0000120b00000001000000010000000000002020200028282800393939005656560071717100a0a0a000c3c3c300f2f2f200e5e5e500bebebe0094949400696969004b4b4b003c3c3c003f3f3f00606060006e6e6e009b9b9b00cdcdcd00dbdbdb00ffffff00adadad007e7e7e006a6a6a00565656004242420032323200575757007b7b7b00979797006f6f6f00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000');
+        done();
+      });
+    });
+  });
+});
+
+
+// //shouldBeGrey:
+// originalMapString = bitmap.toString(54, 1078);
+// if originalMapString.match(/(\d+)\1+\1+/g).length === 256
+//
+
+
+//
+// describe('Buffer Color Palette Conversion Module', function() {
+//   describe('Test for successful inverse conversion', function(done) {
+//     it('which should pass if the color has changed correctly', function(err, data) {
+//     });
+//   });
+// });
+
+  //
+  // shouldBeInvert = `${}`
+  // shouldBeRed = str.match(/255[^255]*/g).length = 256
+  // shouldBeGrey = str.match(/(\d+)\1+\1+/g).length = 256;
+  //
+  // //reverse string d.split('').reverse().join('') Can the same be done on buffer?
+  //
+  // describe('Testing Module Transforms', function() {
+  //   describe('turnBlack switches palette to black', function() {
+  //     it('')
+  //   });
+  // });
```

Comment:
@William-Skelton @shivprogrammer @annaul best to remove unused comment blocks from the `master` branch - these should be contained in a feature or dev branch

overall, excellent tests - you are testing for appropriate outcomes, which is great - only nitpick would be to extract and cluster these tests into 2 files - this one doesn't follow a really cohesive pattern